packages/react-components/stories/omni-panel.stories.tsx

```diff
@@ -79,10 +79,12 @@ const SimpleOmniPanel = (props: SimpleOmniPanelProps): JSX.Element => {
 
 export const SimplePanelFixedSize: Story = (args) => {
   return (
+    // add a background to visualize different theme colors properly
```

Comment:
`rmf-dark` is "broken" with this hardcoded background color.

![image](https://user-images.githubusercontent.com/45189696/125219467-5ec99200-e2f7-11eb-9c5d-cab2630fcb49.png)

Without it, it is even more broken
![image](https://user-images.githubusercontent.com/45189696/125219593-99332f00-e2f7-11eb-82eb-3d788333d56c.png)

I think there is some problem with the theme itself.

Reply:
That's because the omnipanel itself does not have any background color as it is wrap in a div. The background color is passed down from classes in dashboard. For now, I set the background to the same value of what is defined in dashboard

Reply:
Can we fix it by giving it a background color or making it wrapped with a `Paper` instead?